lib/progress_bar.rb

```diff
@@ -1,23 +1,30 @@
-
 require 'options'
 require 'highline'
 
 class ProgressBar
   Error = Class.new(StandardError)
   ArgumentError = Class.new(Error)
 
-  attr_accessor :count, :max, :meters
+  DefaultMeters = [:bar, :counter, :percentage, :elapsed, :eta, :rate]
 
-  def initialize(*args)
+  attr_accessor :count, :max, :meters, :bar, :delimiters
 
+  def initialize(*args, bar: '#', delimiters: '[]')
     @count      = 0
     @max        = 100
-    @meters     = [:bar, :counter, :percentage, :elapsed, :eta, :rate]
 
     @max        = args.shift if args.first.is_a? Numeric
-    raise ArgumentError, "Max must be a positive integer" unless @max >= 0
+    raise ArgumentError, 'Max must be a positive integer' unless @max >= 0
```

Comment:
Seems like this doesn't belong in this PR. In #48 I finally got around to adding rubocop, so feel free to rebase and let it handle the linting.

Reply:
👍 Yeah I noticed my ide was running rubocop. I think it was using a config from another project 🤷‍♂. Ill rebase